UK/Cardiff/St Athan SMR.prf

```diff
@@ -1,4 +1,4 @@
-Settings	SettingsfilePLUGINS	\..\Data\Settings\Local\Cardiff\Plugins.txt
+Settings	SettingsfilePLUGINS	\..\Data\Settings\Local\Cardiff\Plugins_Athans.txt
```

Comment:
?

Reply:
This is to allow me to have a different UKCP config for St Athans than Cardiff. 
i.e. so that St Athans gets overriding go-around alarms from Cardiff.